Nine changes

q4a opened the pull request:
It's draft for Gallium Nine support.

q4a committed:
nine: add option STORM_MESA_NINE for Gallium Nine from Mesa

q4a committed:
nine: don't add SDL_WINDOW_VULKAN to flags if STORM_MESA_NINE defined

espkk commented:
We probably want to have separate conanfiles for linux and windows 🤔maybe after migration to conan 2.0

q4a commented:
I would like to fix bug about trees without fog in Mesa: https://gitlab.freedesktop.org/mesa/mesa/-/issues/8341
Then I'll rebase on top of develop (move Nine support to `cmake/linux.cmake`).
And then it will be ready for merge.

q4a committed:
nine: add support for Gallium Nine as CMake ExternalProject

q4a commented:
I rebased changed on top of develop and moved Nine support to `cmake/linux.cmake`. Looks like Mesa bug will not fixed soon. So I suggest to merge this changes now.
